omod/src/main/webapp/resources/htmlforms/071-IntegratedAntenatalRegister.xml

```diff
@@ -507,6 +509,12 @@
 	            	valid = false;
 	            }
 
+                if (encounterDate &gt;= returnDate) {
+                getField('returndate.error').html('The next appointment date must be after the date of this visit ' +
+					getValue('encounterDate.value')).show();
```

Comment:
@paleu256 Add changeFieldDateToJavascriptDate to format the dates to be consistent with other dates displayed